app/ingestion/jd_mapping.json

```diff
@@ -0,0 +1,43 @@
+{
+  "mappings": {
+    "job_description": {
+      "properties": {
+        "companyName": {
+          "type": "keyword",
+          "store": false,
+          "index": true
+        },
+        "location": {
+          "type": "text",
+          "index": true,
+          "store": true
+        },
+        "scrapeTime": {
+          "type": "date",
+          "format": "epoch_second",
+          "index": true
+        },
+        "summary": {
+          "type": "text",
+          "index": true,
+          "store": true
+        },
+        "title": {
+          "type": "text",
+          "store": false,
+          "index": true,
+          "fields": {
+            "raw": {
+              "type": "keyword"
+            }
+          }
+        },
+        "url": {
+          "type": "keyword",
+          "index": false,
+          "store": true
+        }
```

Comment:
I am planning to collect few more fields (Like when the date when the job was published) - additive changes dont need any reindexing. So, I will probably update these mappings after you merge this.